package.json

```diff
@@ -18,7 +18,8 @@
     "start": "run-s build server",
     "server": "node server",
     "test": "jest --watchAll",
-    "test:CI": "jest"
+    "test:CI": "jest",
+    "docs": "node_modules/.bin/jsdoc -c jsdoc.json"
```

Comment:
💭 Whenever there's a list of things where order doesn't explicitly matter, keeping the list in alphabetical order often makes it easier to find specific entries.  Also, it looks nicer 😁